test/lib/sign.js

```diff
@@ -1,11 +1,23 @@
 'use strict'
 
+const fs = require('fs-extra')
 const path = require('path')
 const mockery = require('mockery')
 
 const ChildProcessMock = require('../fixtures/child_process')
+const sign = require('../../lib/sign')
+const utils = require('../../lib/utils')
 
 describe('Sign', () => {
+  const tmpDir = path.join(require('os').tmpDir(), 'electron-windows-store-cert-test')
```

Comment:
Oops, I think this should be `os.tmpdir()` (the other is [deprecated](https://github.com/nodejs/node/commit/5e5ec2cd1e))